src/App.tsx

```diff
@@ -1,8 +1,17 @@
 import React, {ReactElement} from 'react';
-import {View} from 'react-native';
+import {NavigationContainer} from '@react-navigation/native';
+
+import {MainStack} from './navigation/MainStack';
+import {useNavigationTheme} from './navigation/useNavigationTheme';
 
 function App(): ReactElement | null {
```

Comment:
I usually let TS infer type of the component. I'm not sure if it is possible for this component to return `null`.

```suggestion
function App() {
```

Reply:
this is a default return type of `React.FC`